app/controllers/sessions_controller.rb

```diff
@@ -2,19 +2,19 @@ class SessionsController < ApplicationController
   def new; end
 
   def create
-     @user = User.find_by email: params[:session][:email].downcase
-    if @user&.authenticate params[:session][:password]
+     @user = User.find_by email: params[:sessions][:email].downcase
```

Comment:
why you change `sessions` instead of `session`??

Reply:
because I got error if i use session